### PR TITLE
Cucumber installed along with other gems

### PR DESCRIPTION
**Pull request by rupertlion**
## PT Link

https://www.pivotaltracker.com/story/show/160582257

## Installed cucumber

* Bundle installed
* Added cucumber and database cleaner to gem file
* "Bundle exec cucumber"
* Successfully ran cucumber without tests

## What I learnt

Patience again!

### CODE REVIEW

**Comment by rupertlion** [comment]
We have installed Cucumber - please can we merge?

File: features/support/env.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#
+#   Before('not @no-txn', 'not @selenium', 'not @culerity', 'not @celerity', 'not @javascript') do
+#     DatabaseCleaner.strategy = :transaction
+#   end
+#
+
+# Possible values are :truncation and :transaction
+# The :transaction strategy is faster, but might give you threading problems.
+# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
+Cucumber::Rails::Database.javascript_strategy = :truncation
```
Comment (faraznaeem) [inline features/support/env.rb]: Clean out the comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Comment by faraznaeem** [comment]
add PT link and what you have learned in this PT

**Comment by rupertlion** [comment]
All done

**Pull request merged**